client/components/shipping-rates/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import Dropdown from 'components/dropdown';
+
+const ShippingRates = ( { selectedRates, availableRates, packages, updateValue, dimensionUnit, weightUnit, currencySymbol, errors, layout } ) => {
+	const renderTitle = ( pckg ) => {
+		const dimensions = [ pckg.length, pckg.width, pckg.height ].map( dim => dim + ' ' + dimensionUnit ).join( ' x ' );
+		const weight = pckg.weight + ' ' + weightUnit;
+		return <span>{ dimensions + ' ' + weight }</span>;
```

Comment:
I've always found the `sprintf` syntax quite hard to read. What do you think about ES6 template strings? The `dimensions` string could become like this:
```
const dimensions = `${pckg.length} ${dimensionUnit} x ${pckg.width} ${dimensionUnit} x ${pckg.height} ${dimensionUnit}`;
```